### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by megan07**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+BREAKING CHANGES:
+* bigquery: Add ability to manage credentials to `google_bigquery_connection`.  This field is required as the resource is not useful without them. ([#2111](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2111))
+
+FEATURES:
+* **New Resource:** `data_catalog_tag_template` ([#2120](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2120))
+* **New Resource:** `google_container_analysis_occurence` ([#2114](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2114))
+
+IMPROVEMENTS:
+* appengine: added `inbound_services` to `StandardAppVersion` resource ([#2131](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2131))
+* bigquery: Added `google_bigquery_table` `range_partitioning` to GA ([#2121](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2121))
```
Comment (rileykarson) [inline CHANGELOG.md:11]: Seems GA-only?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
 ## 3.25.0 (Unreleased)
+BREAKING CHANGES:
+* bigquery: Add ability to manage credentials to `google_bigquery_connection`.  This field is required as the resource is not useful without them. ([#2111](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2111))
+
+FEATURES:
+* **New Resource:** `data_catalog_tag_template` ([#2120](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2120))
```
Comment (danawillow) [inline CHANGELOG.md:6]:
```suggestion
* **New Resource:** `google_data_catalog_tag_template` ([#2120](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2120))
```

File: CHANGELOG.md (hunk 1 above)
Comment (danawillow) [inline CHANGELOG.md:11]: This can probably be removed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+* **New Resource:** `google_container_analysis_occurence` ([#2114](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2114))
+
+IMPROVEMENTS:
+* appengine: added `inbound_services` to `StandardAppVersion` resource ([#2131](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2131))
+* bigquery: Added `google_bigquery_table` `range_partitioning` to GA ([#2121](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2121))
+* bigquery: Added support for `google_bigquery_table` `hive_partitioning_options` ([#2121](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2121))
+* container: Changed retry logic for `google_container_node_pool` deletion to use timeouts and retry errors more specifically when cluster is updating. ([#2115](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2115))
```
Comment (danawillow) [inline CHANGELOG.md:13]: This feels like it could be a bugfix, mind taking a look?

**Pull request merged**